lab_1_classify_by_unigrams/main.py

```diff
@@ -37,7 +66,18 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
     :param actual: a list of actual values
     :return: the score
     """
+    if not (
+            isinstance(predicted, list)
+            and isinstance(actual, list)
+            and len(predicted) == len(actual)
+    ):
+        return None
 
+    sum_diff = 0
+    for i, value in enumerate(predicted):
```

Comment:
Consider iterating over both predicted and actual values directly via `zip` function